docker-for-mac/release-notes.md

```diff
@@ -13,12 +13,38 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
-## Docker Desktop 3.4.0
-2021-06-09
+## Docker Desktop 3.5.0
+2021-06-23
 
 [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
 [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
 
+
+### New
+
+- Dev Environments
+
+### Upgrades
+- [Compose V2 beta](https://github.com/docker/compose-cli/releases/tag/v2.0.0-beta.4)
```

Comment:
This should be [Compose V2]

Reply:
It is Compose V2 beta as per https://docs.docker.com/compose/cli-command/.